assignments/lambda-classes.js

```diff
@@ -1 +1,193 @@
 // CODE here for your Lambda Classes
+//************ Base Class ******************
+class Person {
+  constructor(props) {
+    this.name = props.name;
+    this.age = props.age;
+    this.location = props.location;
+    this.gender = props.gender;
+  }
+  speak() {
+    return `Hello my name is ${this.name}, I am from ${this.location}.`;
+  }
+}
+//*************** Instructor and PM *****************
+class Instructor extends Person {
+  constructor(props) {
+    super(props);
+    this.specialty = props.specialty;
+    this.favLanguage = props.favLanguage;
+    this.catchPhrase = props.catchPhrase;
+  }
+  demo(subject) {
+    return `${this.name} says: 'Today we are learning about ${subject}.'`;
+  }
+  grading(student, subject) {
+    function gradeAssignment() {
+      const min = 0;
```

Comment:
I like that you declared the min and max as variables. (Just would be easier to change if we ever wanted to go back, we could just change them in 1 place instead of everywhere they're used)